contrib/platform/src/com/sun/jna/platform/win32/WinUser.java

```diff
@@ -492,6 +518,27 @@ public SIZE(int w, int h) {
     public interface HOOKPROC extends StdCallCallback { }
 
     /**
+     * Defines the message parameters passed to a WH_CALLWNDPROC hook procedure, CallWndProc.
+     */
+    public class CWPSTRUCT extends Structure {
+
+		public CWPSTRUCT(Pointer p) {
+			super(p);
+			//Receiving data and read it from native memory to fill the structure.
+			read();
+		}
+
+		public LPARAM lParam;
+		public WPARAM wParam;
+		public UINT message;
```

Comment:
This should be `public int message`, which would bring the signature in line with the SendMessage and SendMessageTimeout methods.